packages/help-center/src/hooks/use-help-search-query.ts

```diff
@@ -16,7 +16,7 @@ export const useHelpSearchQuery = (
 	const queryClient = useQueryClient();
 
 	return useQuery< SearchResult[] >(
-		[ 'help', search ],
+		[ 'help-center-search', search ],
```

Comment:
For anyone interested, here's the one that was using the same query key:

https://github.com/Automattic/wp-calypso/blob/24bfbebe9da214c118ff24aef93544319154c021/client/data/help/use-help-search-query.ts#L20